### PR TITLE
autocommit after adding or merging changelog

### PR DESCRIPTION
**Pull request by robincartier**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -190,15 +205,22 @@ if (commander.init) {
...
                 if (!error) {
                     fs.unlinkSync(`${fileDir}/${file}`, (error) => { throw error });
                 } else {
                     throw error;
                 }
             });
-        })
+        });
+    }).then(() => {
+        if (config.autoCommit) {
```
Comment (VSerain) [inline index.js]: I think we should have a special config for merge (eg: `config.autoCommitMerge`)

Comment (robincartier) [reply index.js]: Done, I let you resolve if that suits you

**Comment by VSerain** [comment]
I have an error on auto commit, 
I think 'git' package is missing in dependencies (package.json) ! 
image

**Comment by VSerain** [comment]
It's okay for me ! ;)

**Pull request merged**